inductiva/coastal/README.md

```diff
@@ -0,0 +1,86 @@
+### Coastal area
```

Comment:
I think you can add a H1 title "Coastal Dynamics" and then a short paragraph about both simulators and then you can expand on the scenario.